src/main.cpp

```diff
@@ -109,6 +110,9 @@ static bool _get_options(int argc, char **argv, Options &options)
 		("api-token,T", po::value<string>(&options.api_token),
 		 "Use a foundries.io API token for authentication. If not specified, oauth2 will be used")
 
+		("start-daemon", po::value<bool>(&options.start_daemon)->default_value(true),
+		 "Start the aktlualizr-lite systemd service")
```

Comment:
I would just extend this a bit more, like "Start the aktlualizr-lite systemd service automatically after performing the registration."

Reply:
fixed in force-push